### PR TITLE
add langtrace example

### PR DESCRIPTION
**Pull request by yemiadej**

### CODE REVIEW

File: langtrace/langtrace-elastic-demo.py [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+ELASTIC_APM_SERVER_URL = os.getenv("ELASTIC_APM_SERVER_URL")
+SERVICE_NAME = os.getenv("SERVICE_NAME", "langtrace-elastic-demo")
+ELASTIC_APM_SECRET_TOKEN = os.getenv("ELASTIC_APM_SECRET_TOKEN")
+
+# Set up Elastic exporter
+elastic_exporter = OTLPSpanExporter(
+    endpoint=ELASTIC_APM_SERVER_URL,
+    headers={"Authorization": f"Bearer {ELASTIC_APM_SECRET_TOKEN}"}
+)
+
```
Comment (bshetti) [inline langtrace/langtrace-elastic-demo.py:27]: So I wanted to see how to use auto-instrument. Hence do we need to set this here vs leave it set outside of the code?

Comment (bshetti) [reply langtrace/langtrace-elastic-demo.py:27]: setting the exporter and the auth code.

File: langtrace/langtrace-elastic-demo.py [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+# Initialize Langtrace with Elastic exporter
+langtrace.init(
+    custom_remote_exporter=elastic_exporter,
+    batch=True,
+    service_name=SERVICE_NAME
+)
+
```
Comment (bshetti) [inline langtrace/langtrace-elastic-demo.py:33]: Will this langtrace.init work if we set up the OTel env variables outside the code in the shell and run 

opentelemetry-instrument python nameoffile.py

Comment (bshetti) [reply langtrace/langtrace-elastic-demo.py:33]: Hence can I just run 

langtrace.init(
    batch=True,
    service_name=SERVICE_NAME
)

File: langtrace/langtrace-elastic-demo.py (hunk 1 above)
Comment (yemiadej) [reply langtrace/langtrace-elastic-demo.py:27]: Yes you can set this as ENV vars outside of the code.

File: langtrace/langtrace-elastic-demo.py (hunk 2 above)
Comment (yemiadej) [reply langtrace/langtrace-elastic-demo.py:33]: Yes langtrace init will work if you setup OTel ENV variables outside of the code. I will modify the code to show this

Comment (yemiadej) [reply langtrace/langtrace-elastic-demo.py:33]: Hey @bshetti  - I've updated the PR with ENV somewhat outside of code. We are working on making this even better so you don't have to specify the ENV in code at all. Let me know if you run into any issues.

Comment (bshetti) [reply langtrace/langtrace-elastic-demo.py:33]: Thanks. I am going to be modifying the file /tavilyauto-agent-langtrace-option1.py --> that (or the option2) file is what I will be writing a blog from. 

I am going to use this. - `from langtrace_python_sdk import langtrace, with_langtrace_root_span`

vs using:

```
from langtrace_python_sdk import langtrace
from langtrace_python_sdk.utils.with_root_span import with_langtrace_root_span
```

**Review by bshetti — APPROVED** [review]

**Pull request merged**